frontend_v2/src/app/components/challenge/challengediscuss/challengediscuss.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="challenge-card">
+  <div class="ev-card-panel card-bt-margin">
+    <div class="ev-md-container ev-panel-title">
+      <div class="row row-lr-margin">
+        <div class="col-sm-12 col-xs-12 col-lr-pad">
+          <h5 class="fw-light">Discuss</h5>
+          <span class="fs-16">Here you can discuss anything related to the <b>{{challenge.title}}</b>.Get all the anwers
+            for your queries, discuss any new idea or get to know the host team and other participants at the discussion
+            pannel.</span> <br> <br>
+          <div class="row">
+            <div class="col-md-2 col-sm-2 col-xs-6 col-lr-pad forum-btn">
+              <a class="ev-btn btn-waves-effect grad-rec-btn waves-dark ev-btn-dark
```

Comment:
Suggestion: I think here rounded button with border-radius: 20px will look more better than rectangular one and also if we can reduce the padding for this particular button to `10px 30px 10px 30px`. WDYT @Kajol-Kumari

Reply:
yes, @Sanji515 It's looking better than the rectangular one.
![imageedit_2_3092706366](https://user-images.githubusercontent.com/44888949/87219560-e47bd000-c379-11ea-9726-f63f8c7788e2.png)

I have incorporated this. Thanks for suggesting.